Models/Product.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class Product
+    {
+        [Key]
+        public int ProductId { get; set; }
+
+        [Required]
```

Comment:
Okay! So I just learned that the data annotations actually only apply to the properties directly below the annotation. Right now, only `Price`, `CustomerId`, `Quantity`, and `ProductTypeId` are required. If you want all the properties to be required, you'll have to put `[Required]` above every single one.

Reply:
(I'm not saying you actually  _do_ want all of them to be required, just giving an example of how it would be implemented.)